Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP1StateMachine.swift

```diff
@@ -469,6 +469,12 @@ extension HTTPConnectionPool {
             return jitteredBackoff
         }
     }
+
+    struct HTTP2MigrationContext {
```

Comment:
we probably should be more explicit with the name `HTTP2ToHTTP1MigrationContext`